lib/policyEvaluator/utils/actionMaps.js

```diff
@@ -30,6 +30,7 @@ const sharedActionMap = {
     bypassGovernanceRetention: 's3:BypassGovernanceRetention',
     listMultipartUploads: 's3:ListBucketMultipartUploads',
     listParts: 's3:ListMultipartUploadParts',
+    metadataSearch: 's3:MetadataSearch',
```

Comment:
In the design we have `s3:mdsearch`, is it expected to use this format now?

Reply:
wanted to ask your suggestion on this, feel like in action names it's not common to have abbreviation, `s3: MDSearch` seems a little bit weird here
@anurag4DSB  didn't think it through when we decided on this name, can we change the action name defined in the design?

Reply:
IMO as long as all implementations are compliant with this name and the design is updated in a quick PR, that's ok